tasks/scripts.js

```diff
@@ -8,7 +10,7 @@ module.exports = (gulp, $, pkg) => {
       default: { concat: true },
     }), args);
     return gulp.src(pkg.gulpPaths.scripts.src)
-      .pipe($.plumber())
+      .pipe(process.env.GULP_ABORT_ON_ERROR ? noop() : $.plumber())
```

Comment:
This appears to be the same as the built-in errorHandler option – do we need the additional dependency?
https://www.npmjs.com/package/gulp-plumber#user-content-optionserrorhandler

```suggestion
      .pipe($.plumber({ errorHandler: !process.env.GULP_ABORT_ON_ERROR }))
```

Reply:
You are right @sun, the additional dependency is not needed. I was event able to remove plumber altogether, the same behavior can be achieved by using the options for gulp-stylelint and gulp-eslint. Please see my new commit.